services/src/main/scala/cromwell/services/metadata/WorkflowQueryParameters.scala

```diff
@@ -84,7 +86,8 @@ object WorkflowQueryParameters {
       startDateValidation,
       endDateValidation,
       pageValidation,
-      pageSizeValidation) mapN { (_, _, statuses, names, ids, labels, startDate, endDate, page, pageSize) => WorkflowQueryParameters(statuses, names, ids, labels, startDate, endDate, page, pageSize) }
+      pageSizeValidation,
+      additionalKeysValidation) mapN { (_, _, statuses, names, ids, labels, startDate, endDate, page, pageSize, additionalKeys) => WorkflowQueryParameters(statuses, names, ids, labels, startDate, endDate, page, pageSize, additionalKeys) }
```

Comment:
~~Is this equivalent to `...) mapN WorkflowQueryParameters`?~~
EDIT: never mind, I missed the two `_`s at the start!